members/khoi2109.md

```diff
@@ -0,0 +1,34 @@
+# Khoi
+
+![](https://avatars.githubusercontent.com/u/48615950)
+
+## Position
+- Intern (01/2024 - Now)
+
+## Basic Info
+- **DoB**: 21/09/2003
+- **Gender**: Male
+- **Phone number**: 0984146006
+- **Email**: khoikhtn2109@gmail.com
+- **Facebook**: [Minh Khoi Trinh](https://www.facebook.com/khoispiderman/)
+
+## Intro
+
+### For New Members
+Hello everyone, I'm Khoi. Feel free to reach me
+
+### For Our Team
+- I will contribute to this group to the best of my ability.
+- And also want to learn from you guys, I am not a fast learner, but rather a hard working one.
+
+### If You Want to Hear, I Could Tell You
+- Chat
+
+### In a Rainy Day, You Could Make Me Smile By
+- Play Taylor Swift's Music
```

Comment:
I think "playing" is more correct in this context.

Reply:
Yeah my bad